nixos/modules/profiles/macos-builder.nix

```diff
@@ -234,6 +234,10 @@ in
       # This ensures that anything built on the guest isn't lost when the guest is
       # restarted.
       writableStoreUseTmpfs = false;
+
+      # Pass certificates from host to the guest otherwise we run into a bootstrapping
+      # problem when we need custom CA certificates.
```

Comment:
As I was reading this comment, I was asking myself what does this bootstrap problem entail and when might I run into it. If this comment is worth documenting, would it be possible to expand upon it?

Reply:
Updated the comment :+1: